src/Modules/CalcDefence.lua

```diff
@@ -2349,8 +2351,10 @@ function calcs.defence(env, actor)
 			return receivedDamageSum
 		end
 
+		local enemyDamageMult = output[damageType .."EnemyDamageMult"]
+
 		if partMin == m_huge then
-			output[damageType.."MaximumHitTaken"] = m_huge
+			output[damageType.."MaximumHitTaken"] = m_huge / enemyDamageMult
```

Comment:
```suggestion
			output[damageType.."MaximumHitTaken"] = m_huge
```
No point poking m_huge